signing/watch/notify_corp_admin.go

```diff
@@ -120,16 +120,17 @@ func (impl *notifyAdminWatchImpl) handleCorpSigning(link *repository.LinkCLA, co
 }
 
 func (impl *notifyAdminWatchImpl) isCorpSigningLatest(latestCLAs []domain.CLA, signedInfo domain.CLAInfo) bool {
+	var matchedCLA domain.CLA
+
 	for i := range latestCLAs {
 		if latestCLAs[i].Type == dp.CLATypeCorp &&
-			latestCLAs[i].Language == signedInfo.Language &&
-			latestCLAs[i].Id == signedInfo.CLAId {
-
-			return true
+			latestCLAs[i].Language == signedInfo.Language {
+			matchedCLA = latestCLAs[i]
+			break
 		}
 	}
 
-	return false
+	return matchedCLA.Id == signedInfo.CLAId
```

Comment:
var matchedCLA *domain.CLA

	for i := range latestCLAs {
		if latestCLAs[i].Type == dp.CLATypeCorp &&
			latestCLAs[i].Language == signedInfo.Language {
			matchedCLA = &latestCLAs[i]
			break
		}
	}

	return matchedCLA != nil &&  matchedCLA.Id == signedInfo.CLAId